lta/rest_server.py

```diff
@@ -286,15 +288,18 @@ class BundlesActionsPopHandler(BaseLTAHandler):
         query = {
             "$and": [
                 {"location": {"$regex": f"^{site}"}},
+                {"status": f"{status}"},
                 {"$or": [
                     {"claimed": False},
                     {"claim_time": {"$lt": f"{old_age}"}}
                 ]}
             ]
         }
+        sort = [('create_timestamp', ASCENDING)]
         ret = []
         async for row in sdb.find(filter=query,
-                                  projection=REMOVE_ID):
+                                  projection=REMOVE_ID,
+                                  sort=sort):
```

Comment:
Since I just went through this for IceProd, make sure there's an index for that sort.

Reply:
Added for both Files and Bundles:
https://github.com/WIPACrepo/lta/pull/25/commits/fa01a201a67d3eb14cfd1043c8f54409b58007e2